index.html

```diff
@@ -10,18 +10,22 @@
   
     <div id="mainContainer">
   <div id="headerDiv">
+    <div id="BtnsDiv"><button id="newGridBut">New Grid</button>
+       <button id="clearBut">Clear</button> 
+   
+      <input type="checkbox" id='erase' class='chk-btn' />
+  <label for='erase'>Erase</label>
+  <br id="break"/><div id="together">
+  <input type="checkbox" id="makeBlack" ><label for="makeBlack"> Darken colors</label></input></div>
+  </div>
 <div id="title"><h1>Etch A Sketch</h1></div>
     <div class="radioBtns">
-      <input type="checkbox" id="makeBlack" >Darken colors</input> &nbsp;&nbsp;&nbsp;  &nbsp;&nbsp;&nbsp;  <strong>Mode: </strong> &nbsp; 
-    <input type="radio" name="mode" id="radioBut1RandomColors" value="randomClrs" checked="checked">Random Colors</input>
-    <input type="radio" name="mode" id="radioBut2RandomColor" value="randomClr">Random Color &nbsp;  <div id="clrSelected"></div></input>
-    <input type="radio" name="mode" id="radioBut3PickColor" value="pikClr">Pick Color<input type="color" id="clr"></input></input>
+       <br> &nbsp;&nbsp;&nbsp;  <strong>Mode: </strong> &nbsp;&nbsp; 
+    <input type="radio" name="mode" id="radioBut1RandomColors" value="randomClrs" checked="checked"><label for="radioBut1RandomColors">&nbsp;Random Colors</label> </input>
+    <span class="spaces">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span>&nbsp;&nbsp;&nbsp;<div id="together"><input type="radio" name="mode" id="radioBut2RandomColor" value="randomClr"><label for="radioBut2RandomColor">&nbsp;Random Color</label>  <div id="clrSelected"></div></input></div>
+    <span class="spaces">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span>&nbsp;&nbsp;&nbsp;<div id="together"><input type="radio" name="mode" id="radioBut3PickColor" value="pikClr"><label for="radioBut3PickColor">&nbsp;Pick Color</label></input>&nbsp;<input type="color" id="clr"></input></div>
   </div>
```

Comment:
One last thing which is **dangerous** to keep in your code, each element must has a unique Id, you named three divs with the same <id="together"/> which is affect your accessibility as well. 
PS: changing the [id] is amust. but adding the label for your form is up to you to increase your score of accessibility.
![image](https://user-images.githubusercontent.com/56412800/98460959-5f7eb700-21b1-11eb-87b6-b50bee767f6c.png)